app.rb

```diff
@@ -19,29 +19,6 @@ class << self
   API_PREFIX = "/api/#{API_VERSION}"
 end
 
-if ["staging", "production", "loadtest", "edgestage","edgeprod"].include? environment
-  require 'newrelic_rpm'
-  require 'new_relic/agent/method_tracer'
-  Moped::Session.class_eval do
-    include NewRelic::Agent::MethodTracer
-    add_method_tracer :new
-    add_method_tracer :use
-    add_method_tracer :login
-  end
-  Moped::Cluster.class_eval do
-    include NewRelic::Agent::MethodTracer
-    add_method_tracer :with_primary
-    add_method_tracer :nodes
-  end
-  Moped::Node.class_eval do
-    include NewRelic::Agent::MethodTracer
-    add_method_tracer :command
-    add_method_tracer :connect
-    add_method_tracer :flush
-    add_method_tracer :refresh
-  end
-end
```

Comment:
@smarnach Thanks for reminding me about newrelic integration. I pushed a commit that removes the old `newrelic_mongodb` gem and updated `newrelic_rpm` to the most recent version. `newrelic_rpm` now has [bulit-in support for `mongo` ruby driver](https://docs.newrelic.com/docs/agents/ruby-agent/frameworks/mongo-instrumentation).

Moped is no longer used and the new mongo driver codebase is different enough from moped that I wouldn't be able to instrument the exact same methods. I think it makes sense to start with the metrics that the new `newrelic_rpm` gem provides by default, and add more detailed instrumentation using `add_method_tracer` at a later point if it turns out the default metrics don't provide enough insight.